src/plugins/regions.ts

```diff
@@ -90,8 +92,9 @@ class SingleRegion extends EventEmitter<RegionEvents> {
   private initElement(content?: string | HTMLElement) {
     const element = document.createElement('div')
     const isMarker = this.start === this.end
-
-    element.setAttribute('part', `${isMarker ? 'marker' : 'region'} ${this.id}`)
+  
+    element.setAttribute('part', `${isMarker ? 'marker' : 'region'}`)
+    element.setAttribute('id', Array.isArray(this.id) ? this.id.join(' ') : this.id)
```

Comment:
Adding an id won't be useful, as it's part of a Shadow DOM tree. Only the part attribute is usable for CSS styling. So please keep it as it was.